pkgs/epnix/support/autoparamDriver/default.nix

```
@@ -0,0 +1,31 @@
+{
+  mkEpicsPackage,
+  fetchFromGitHub,
+  epnix,
+  lib,
+  epnixLib,
+}:
+
+mkEpicsPackage rec {
+  pname = "autoparamDriver";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "Cosylab";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-J2fy/pMwrbwVFULfANuJBl6iE3wju5bQkhkxxk8zRYs=";
+  };
+
+  propagatedBuildInputs = with epnix.support; [ asyn ];
+
+  # TODO: is this correct?
+  varname = "APD";
```

I see that adsDriver uses `AUTOPARAM` so we can use that

```suggestion
  varname = "AUTOPARAM";
```